web/src/lib/components/forms/tag-asset-form.svelte

```diff
@@ -0,0 +1,52 @@
+<script lang="ts">
+  import { mdiTag } from '@mdi/js';
+  import { t } from 'svelte-i18n';
+  import Button from '../elements/buttons/button.svelte';
+  import Combobox, { type ComboBoxOption } from '../shared-components/combobox.svelte';
+  import FullScreenModal from '../shared-components/full-screen-modal.svelte';
+  import { onMount } from 'svelte';
+  import { getAllTags, type TagResponseDto } from '@immich/sdk';
+
+  export let onTag: (tagIds: string[]) => void;
+  export let onCancel: () => void;
+
+  let tags: TagResponseDto[] = [];
+  let selectedTags = new Set<string>();
+  $: disabled = selectedTags.size === 0;
+
+  onMount(async () => {
+    tags = await getAllTags();
+  });
+
+  const handleSubmit = () => {
+    onTag([...selectedTags]);
+    selectedTags.clear();
+  };
+
+  const handleSelect = (option?: ComboBoxOption) => {
+    if (!option) {
+      selectedTags = new Set();
+      return;
+    }
+
+    selectedTags.add(option.value);
+    selectedTags = new Set(selectedTags);
```

Comment:
Why do we create a copy here? To trigger reactivity we can also write `selectedTags = selectedTags`, right?